internal/install/recipe_installer.go

```diff
@@ -288,14 +290,45 @@ func (i *RecipeInstall) install(ctx context.Context) error {
 	return nil
 }
 
+func (i *RecipeInstall) printStartInstallingMessage(repo *recipes.RecipeRepository) {
+	message := "\n\nInstalling New Relic"
+	if i.RecipeNamesProvided() && len(i.RecipeNames) > 0 {
+		r := repo.FindRecipeByName(i.RecipeNames[0])
+		if r != nil {
+			message = fmt.Sprintf("%s %s", message, r.DisplayName)
+		}
+	}
+	fmt.Println(message)
+}
+
+func (i *RecipeInstall) reportRecipeStatuses(availableRecipes map[string]*recipes.RecipeDetectionResult,
+	unavailableRecipes map[string]*recipes.RecipeDetectionResult) {
+
+	for _, d := range unavailableRecipes {
+		e := execution.RecipeStatusEvent{Recipe: *d.Recipe, ValidationDurationMs: d.DurationMs}
```

Comment:
@NRhzhao I'm not sure we'd want to do that. This will insert an event (which status?) for every recipe that doesn't detect? Or is this reporting the recipes that are detected but not available? If those are the "detected/not available recipes" perhaps we could rename unavailableRecipes to detectedRecipes, or detectedUnavailableRecipes?

Reply:
It doesn't report the status of the recipes that are not detected.  I believe this is the same way we did detection for the guided install before.  I double-checked the log, and it doesn't produce any extra events.
This doesn't just report unavailable recipes, it also reports available recipes.  The use case is if a person does a target install for PHP, and we detected MySQL on the same host and status is available.  MySQL would be part of the available recipes.  So we want to also report the available recipes in case, so we just report detect status here for datagap.  During installation, the other events for the recipe that's installed will be handled correctly.

Reply:
TY